### ISSUE TITLE
npm ERR! peerinvalid The package classnames does not satisfy its siblings' peerDependencies requirements!

### ISSUE THREAD

**Issue by io3** [tomchentw/react-toastr#32]
```
npm ERR! node v0.12.2
npm ERR! npm  v2.7.6
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package classnames does not satisfy its siblings' peerDependencies requirements!
```

Too many packages require `classnames`, why peerDependencies ?

```
"peerDependencies": {
    "classnames": "^2.1.2"
  }
```

**Comment by tomchentw** [comment]
Could I downgrade it to `^2.0.0`?

**Comment by io3** [comment]
How about move classnames to `dependencies` ?

**Comment by tomchentw** [comment]
Released [v2.1.0](https://github.com/tomchentw/react-toastr/releases/tag/v2.1.0)

**Comment by luqin** [comment]
:+1:
